runCompareTwoGenomesColinear.sh

```diff
@@ -16,23 +16,39 @@ if [ $# -ne 11 ]; then
         exit 1
 fi
 
-echo Running: $(basename $0) $1 $2 $3 $4 $5 $6 $7 $8 $9 ${10} ${11}
+if [ ${11} == '.' ]; then
```

Comment:
What do you think about adding a comment to the 'help' printout about using `.` when the user doesn't want to specify the optional arguments? Just thinking that might not be obvious to all.

Reply:
I've added some comments to the help section